src/Shared/AsyncWebView/AsyncWebView.Properties.cs

```diff
@@ -146,7 +142,7 @@ public string SourceHtml
 		/// Source message property
 		/// </summary>
 		public static readonly DependencyProperty SourceMessageProperty =
-			DependencyProperty.Register("SourceMessage", typeof(Windows.Web.Http.HttpRequestMessage), typeof(AsyncWebView), new PropertyMetadata(null, (s, e) => (s as AsyncWebView)?.OnSourceMessageChanged(e.NewValue as Windows.Web.Http.HttpRequestMessage)));
+			DependencyProperty.Register("SourceMessage", typeof(HttpRequestMessage), typeof(AsyncWebView), new PropertyMetadata(null, (s, e) => (s as AsyncWebView)?.OnSourceMessageChanged(e.NewValue as HttpRequestMessage)));
```

Comment:
The current error on the pipeline is the reason why I had to be more specific on the difference between `System.Net.Http.HttpRequestMessage` and `Windows.Web.Http.HttpRequestMessage`, because OnSourceMessageChanged for the Uno AsyncWebView still needs `Windows.Web.Http`. Maybe this file could be seperated instead of shared or we could add some conditions here